legend-engine-pure-code-compiled-core/src/main/resources/core/pure/graphFetch/graphFetch.pure

```diff
@@ -54,7 +54,8 @@ function meta::pure::graphFetch::execution::graphFetchFunctions():Function<Any>[
     meta::pure::graphFetch::execution::graphFetch_T_MANY__RootGraphFetchTree_1__Integer_1__T_MANY_, 
     meta::pure::graphFetch::execution::graphFetchChecked_T_MANY__RootGraphFetchTree_1__Checked_MANY_, 
     meta::pure::graphFetch::execution::graphFetchUnexpanded_T_MANY__RootGraphFetchTree_1__T_MANY_,
-    meta::pure::graphFetch::execution::graphFetchCheckedUnexpanded_T_MANY__RootGraphFetchTree_1__Checked_MANY_
+    meta::pure::graphFetch::execution::graphFetchCheckedUnexpanded_T_MANY__RootGraphFetchTree_1__Checked_MANY_,
+    meta::pure::graphFetch::execution::serialize_T_MANY__RootGraphFetchTree_1__String_1_
```

Comment:
do we need this?  This looks like should be part of core & wondering how the existing queries work??

Reply:
indeed we don't, not sure why I added this, maybe I tried to execute a wrong query at some point and gave me a misleading error

Reply:
ok, thx. will merge the rest and revert this particular change